Test failure with python 3.11b3

Fedora has updated to Python 3.11b3.  toolz has a patch but cytoolz fails with:
```
+ py.test-3.11 cytoolz/tests -k 'not test_introspect_builtin_modules' -v
============================= test session starts ==============================
platform linux -- Python 3.11.0b3, pytest-7.1.2, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /home/orion/fedora/python-cytoolz/cytoolz-0.11.2
plugins: remotedata-0.3.3, doctestplus-0.11.2, forked-1.4.0, rerunfailures-10.2, flake8-1.1.1, timeout-2.1.0, xdist-2.5.0, mock-3.7.0, arraydiff-0.4.0, pyfakefs-4.5.6, openfiles-0.5.0, mpi-0.6
collecting ... collected 194 items / 1 deselected / 193 selected

cytoolz/tests/test_compatibility.py::test_compat_warn PASSED             [  0%]
cytoolz/tests/test_curried.py::test_take PASSED                          [  1%]
cytoolz/tests/test_curried.py::test_first PASSED                         [  1%]
cytoolz/tests/test_curried.py::test_merge PASSED                         [  2%]
cytoolz/tests/test_curried.py::test_merge_with PASSED                    [  2%]
cytoolz/tests/test_curried.py::test_merge_with_list PASSED               [  3%]
cytoolz/tests/test_curried.py::test_sorted PASSED                        [  3%]
cytoolz/tests/test_curried.py::test_reduce PASSED                        [  4%]
cytoolz/tests/test_curried.py::test_module_name PASSED                   [  4%]
cytoolz/tests/test_curried.py::test_curried_operator PASSED              [  5%]
cytoolz/tests/test_curried.py::test_curried_namespace PASSED             [  5%]
cytoolz/tests/test_curried_toolzlike.py::test_toolzcurry_is_class PASSED [  6%]
cytoolz/tests/test_curried_toolzlike.py::test_cytoolz_like_toolz PASSED  [  6%]
cytoolz/tests/test_curried_toolzlike.py::test_toolz_like_cytoolz PASSED  [  7%]
cytoolz/tests/test_dev_skip_test.py::test_passes PASSED                  [  7%]
cytoolz/tests/test_dev_skip_test.py::test_dev_skip_test PASSED           [  8%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_merge PASSED             [  8%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_merge_iterable_arg PASSED [  9%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_merge_with PASSED        [  9%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_merge_with_iterable_arg PASSED [ 10%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_valmap PASSED            [ 10%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_keymap PASSED            [ 11%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_itemmap PASSED           [ 11%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_valfilter PASSED         [ 12%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_keyfilter PASSED         [ 12%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_itemfilter PASSED        [ 13%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_assoc PASSED             [ 13%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_dissoc PASSED            [ 14%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_assoc_in PASSED          [ 15%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_update_in PASSED         [ 15%]
cytoolz/tests/test_dicttoolz.py::TestDict::test_factory PASSED           [ 16%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_merge PASSED      [ 16%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_merge_iterable_arg PASSED [ 17%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_merge_with PASSED [ 17%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_merge_with_iterable_arg PASSED [ 18%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_valmap PASSED     [ 18%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_keymap PASSED     [ 19%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_itemmap PASSED    [ 19%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_valfilter PASSED  [ 20%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_keyfilter PASSED  [ 20%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_itemfilter PASSED [ 21%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_assoc PASSED      [ 21%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_dissoc PASSED     [ 22%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_assoc_in PASSED   [ 22%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_update_in PASSED  [ 23%]
cytoolz/tests/test_dicttoolz.py::TestDefaultDict::test_factory PASSED    [ 23%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_merge PASSED    [ 24%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_merge_iterable_arg PASSED [ 24%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_merge_with PASSED [ 25%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_merge_with_iterable_arg PASSED [ 25%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_valmap PASSED   [ 26%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_keymap PASSED   [ 26%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_itemmap PASSED  [ 27%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_valfilter PASSED [ 27%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_keyfilter PASSED [ 28%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_itemfilter PASSED [ 29%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_assoc PASSED    [ 29%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_dissoc PASSED   [ 30%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_assoc_in PASSED [ 30%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_update_in PASSED [ 31%]
cytoolz/tests/test_dicttoolz.py::TestCustomMapping::test_factory PASSED  [ 31%]
cytoolz/tests/test_dicttoolz.py::test_environ PASSED                     [ 32%]
cytoolz/tests/test_dicttoolz.py::test_merge_with_non_dict_mappings PASSED [ 32%]
cytoolz/tests/test_docstrings.py::test_docstrings_uptodate PASSED        [ 33%]
cytoolz/tests/test_docstrings.py::test_get_in_doctest PASSED             [ 33%]
cytoolz/tests/test_doctests.py::test_doctest PASSED                      [ 34%]
cytoolz/tests/test_embedded_sigs.py::test_class_sigs FAILED              [ 34%]
cytoolz/tests/test_embedded_sigs.py::test_sig_at_beginning PASSED        [ 35%]
cytoolz/tests/test_functoolz.py::test_apply PASSED                       [ 35%]
cytoolz/tests/test_functoolz.py::test_thread_first PASSED                [ 36%]
cytoolz/tests/test_functoolz.py::test_thread_last PASSED                 [ 36%]
cytoolz/tests/test_functoolz.py::test_memoize PASSED                     [ 37%]
cytoolz/tests/test_functoolz.py::test_memoize_kwargs PASSED              [ 37%]
cytoolz/tests/test_functoolz.py::test_memoize_curried PASSED             [ 38%]
cytoolz/tests/test_functoolz.py::test_memoize_partial PASSED             [ 38%]
cytoolz/tests/test_functoolz.py::test_memoize_key_signature PASSED       [ 39%]
cytoolz/tests/test_functoolz.py::test_memoize_curry_cache PASSED         [ 39%]
cytoolz/tests/test_functoolz.py::test_memoize_key PASSED                 [ 40%]
cytoolz/tests/test_functoolz.py::test_memoize_wrapped PASSED             [ 40%]
cytoolz/tests/test_functoolz.py::test_curry_simple PASSED                [ 41%]
cytoolz/tests/test_functoolz.py::test_curry_kwargs PASSED                [ 41%]
cytoolz/tests/test_functoolz.py::test_curry_passes_errors PASSED         [ 42%]
cytoolz/tests/test_functoolz.py::test_curry_docstring PASSED             [ 43%]
cytoolz/tests/test_functoolz.py::test_curry_is_like_partial PASSED       [ 43%]
cytoolz/tests/test_functoolz.py::test_curry_is_idempotent PASSED         [ 44%]
cytoolz/tests/test_functoolz.py::test_curry_attributes_readonly PASSED   [ 44%]
cytoolz/tests/test_functoolz.py::test_curry_attributes_writable PASSED   [ 45%]
cytoolz/tests/test_functoolz.py::test_curry_module PASSED                [ 45%]
cytoolz/tests/test_functoolz.py::test_curry_comparable PASSED            [ 46%]
cytoolz/tests/test_functoolz.py::test_curry_doesnot_transmogrify PASSED  [ 46%]
cytoolz/tests/test_functoolz.py::test_curry_on_classmethods PASSED       [ 47%]
cytoolz/tests/test_functoolz.py::test_memoize_on_classmethods PASSED     [ 47%]
cytoolz/tests/test_functoolz.py::test_curry_call PASSED                  [ 48%]
cytoolz/tests/test_functoolz.py::test_curry_bind PASSED                  [ 48%]
cytoolz/tests/test_functoolz.py::test_curry_unknown_args PASSED          [ 49%]
cytoolz/tests/test_functoolz.py::test_curry_bad_types PASSED             [ 49%]
cytoolz/tests/test_functoolz.py::test_curry_subclassable PASSED          [ 50%]
cytoolz/tests/test_functoolz.py::test_compose PASSED                     [ 50%]
cytoolz/tests/test_functoolz.py::test_compose_metadata PASSED            [ 51%]
cytoolz/tests/test_functoolz.py::test_compose_left PASSED                [ 51%]
cytoolz/tests/test_functoolz.py::test_pipe PASSED                        [ 52%]
cytoolz/tests/test_functoolz.py::test_complement PASSED                  [ 52%]
cytoolz/tests/test_functoolz.py::test_do PASSED                          [ 53%]
cytoolz/tests/test_functoolz.py::test_juxt_generator_input PASSED        [ 53%]
cytoolz/tests/test_functoolz.py::test_flip PASSED                        [ 54%]
cytoolz/tests/test_functoolz.py::test_excepts PASSED                     [ 54%]
cytoolz/tests/test_inspect_args.py::test_make_func PASSED                [ 55%]
cytoolz/tests/test_inspect_args.py::test_is_valid PASSED                 [ 55%]
cytoolz/tests/test_inspect_args.py::test_is_valid_py3 PASSED             [ 56%]
cytoolz/tests/test_inspect_args.py::test_is_partial PASSED               [ 56%]
cytoolz/tests/test_inspect_args.py::test_is_valid_curry PASSED           [ 57%]
cytoolz/tests/test_inspect_args.py::test_func_keyword PASSED             [ 58%]
cytoolz/tests/test_inspect_args.py::test_has_unknown_args PASSED         [ 58%]
cytoolz/tests/test_inspect_args.py::test_num_required_args PASSED        [ 59%]
cytoolz/tests/test_inspect_args.py::test_has_keywords PASSED             [ 59%]
cytoolz/tests/test_inspect_args.py::test_has_varargs PASSED              [ 60%]
cytoolz/tests/test_inspect_args.py::test_is_arity PASSED                 [ 60%]
cytoolz/tests/test_inspect_args.py::test_introspect_curry_valid_py3 PASSED [ 61%]
cytoolz/tests/test_inspect_args.py::test_introspect_curry_partial_py3 PASSED [ 61%]
cytoolz/tests/test_inspect_args.py::test_introspect_curry_py3 PASSED     [ 62%]
cytoolz/tests/test_inspect_args.py::test_inspect_signature_property PASSED [ 62%]
cytoolz/tests/test_inspect_args.py::test_inspect_wrapped_property PASSED [ 63%]
cytoolz/tests/test_itertoolz.py::test_remove PASSED                      [ 63%]
cytoolz/tests/test_itertoolz.py::test_groupby PASSED                     [ 64%]
cytoolz/tests/test_itertoolz.py::test_groupby_non_callable PASSED        [ 64%]
cytoolz/tests/test_itertoolz.py::test_merge_sorted PASSED                [ 65%]
cytoolz/tests/test_itertoolz.py::test_interleave PASSED                  [ 65%]
cytoolz/tests/test_itertoolz.py::test_unique PASSED                      [ 66%]
cytoolz/tests/test_itertoolz.py::test_isiterable PASSED                  [ 66%]
cytoolz/tests/test_itertoolz.py::test_isdistinct PASSED                  [ 67%]
cytoolz/tests/test_itertoolz.py::test_nth PASSED                         [ 67%]
cytoolz/tests/test_itertoolz.py::test_first PASSED                       [ 68%]
cytoolz/tests/test_itertoolz.py::test_second PASSED                      [ 68%]
cytoolz/tests/test_itertoolz.py::test_last PASSED                        [ 69%]
cytoolz/tests/test_itertoolz.py::test_rest PASSED                        [ 69%]
cytoolz/tests/test_itertoolz.py::test_take PASSED                        [ 70%]
cytoolz/tests/test_itertoolz.py::test_tail PASSED                        [ 70%]
cytoolz/tests/test_itertoolz.py::test_drop PASSED                        [ 71%]
cytoolz/tests/test_itertoolz.py::test_take_nth PASSED                    [ 72%]
cytoolz/tests/test_itertoolz.py::test_get PASSED                         [ 72%]
cytoolz/tests/test_itertoolz.py::test_mapcat PASSED                      [ 73%]
cytoolz/tests/test_itertoolz.py::test_cons PASSED                        [ 73%]
cytoolz/tests/test_itertoolz.py::test_concat PASSED                      [ 74%]
cytoolz/tests/test_itertoolz.py::test_concatv PASSED                     [ 74%]
cytoolz/tests/test_itertoolz.py::test_interpose PASSED                   [ 75%]
cytoolz/tests/test_itertoolz.py::test_frequencies PASSED                 [ 75%]
cytoolz/tests/test_itertoolz.py::test_reduceby PASSED                    [ 76%]
cytoolz/tests/test_itertoolz.py::test_reduce_by_init PASSED              [ 76%]
cytoolz/tests/test_itertoolz.py::test_reduce_by_callable_default PASSED  [ 77%]
cytoolz/tests/test_itertoolz.py::test_iterate PASSED                     [ 77%]
cytoolz/tests/test_itertoolz.py::test_accumulate PASSED                  [ 78%]
cytoolz/tests/test_itertoolz.py::test_accumulate_works_on_consumable_iterables PASSED [ 78%]
cytoolz/tests/test_itertoolz.py::test_sliding_window PASSED              [ 79%]
cytoolz/tests/test_itertoolz.py::test_sliding_window_of_short_iterator PASSED [ 79%]
cytoolz/tests/test_itertoolz.py::test_partition PASSED                   [ 80%]
cytoolz/tests/test_itertoolz.py::test_partition_all PASSED               [ 80%]
cytoolz/tests/test_itertoolz.py::test_count PASSED                       [ 81%]
cytoolz/tests/test_itertoolz.py::test_pluck PASSED                       [ 81%]
cytoolz/tests/test_itertoolz.py::test_join PASSED                        [ 82%]
cytoolz/tests/test_itertoolz.py::test_getter PASSED                      [ 82%]
cytoolz/tests/test_itertoolz.py::test_key_as_getter PASSED               [ 83%]
cytoolz/tests/test_itertoolz.py::test_join_double_repeats PASSED         [ 83%]
cytoolz/tests/test_itertoolz.py::test_join_missing_element PASSED        [ 84%]
cytoolz/tests/test_itertoolz.py::test_left_outer_join PASSED             [ 84%]
cytoolz/tests/test_itertoolz.py::test_right_outer_join PASSED            [ 85%]
cytoolz/tests/test_itertoolz.py::test_outer_join PASSED                  [ 86%]
cytoolz/tests/test_itertoolz.py::test_diff PASSED                        [ 86%]
cytoolz/tests/test_itertoolz.py::test_topk PASSED                        [ 87%]
cytoolz/tests/test_itertoolz.py::test_topk_is_stable PASSED              [ 87%]
cytoolz/tests/test_itertoolz.py::test_peek PASSED                        [ 88%]
cytoolz/tests/test_itertoolz.py::test_peekn PASSED                       [ 88%]
cytoolz/tests/test_itertoolz.py::test_random_sample PASSED               [ 89%]
cytoolz/tests/test_none_safe.py::test_dicttoolz PASSED                   [ 89%]
cytoolz/tests/test_none_safe.py::test_functoolz PASSED                   [ 90%]
cytoolz/tests/test_none_safe.py::test_itertoolz PASSED                   [ 90%]
cytoolz/tests/test_none_safe.py::test_recipes PASSED                     [ 91%]
cytoolz/tests/test_recipes.py::test_countby PASSED                       [ 91%]
cytoolz/tests/test_recipes.py::test_partitionby PASSED                   [ 92%]
cytoolz/tests/test_serialization.py::test_compose PASSED                 [ 92%]
cytoolz/tests/test_serialization.py::test_curry PASSED                   [ 93%]
cytoolz/tests/test_serialization.py::test_juxt PASSED                    [ 93%]
cytoolz/tests/test_serialization.py::test_complement PASSED              [ 94%]
cytoolz/tests/test_serialization.py::test_instanceproperty PASSED        [ 94%]
cytoolz/tests/test_serialization.py::test_flip PASSED                    [ 95%]
cytoolz/tests/test_serialization.py::test_curried_exceptions PASSED      [ 95%]
cytoolz/tests/test_serialization.py::test_curried_qualname PASSED        [ 96%]
cytoolz/tests/test_serialization.py::test_curried_bad_qualname PASSED    [ 96%]
cytoolz/tests/test_signatures.py::test_is_valid PASSED                   [ 97%]
cytoolz/tests/test_signatures.py::test_is_partial PASSED                 [ 97%]
cytoolz/tests/test_signatures.py::test_for_coverage PASSED               [ 98%]
cytoolz/tests/test_tlz.py::test_tlz PASSED                               [ 98%]
cytoolz/tests/test_utils.py::test_raises PASSED                          [ 99%]
cytoolz/tests/test_utils.py::test_consume PASSED                         [100%]

=================================== FAILURES ===================================
_______________________________ test_class_sigs ________________________________

    @dev_skip_test
    def test_class_sigs():
        """ Test that all ``cdef class`` extension types in ``cytoolz`` have
            correctly embedded the function signature as done in ``toolz``.
        """
        import toolz
        # only consider items created in both `toolz` and `cytoolz`
        toolz_dict = valfilter(isfrommod('toolz'), toolz.__dict__)
        cytoolz_dict = valfilter(isfrommod('cytoolz'), cytoolz.__dict__)
    
        # only test `cdef class` extensions from `cytoolz`
        cytoolz_dict = valfilter(lambda x: not isinstance(x, BuiltinFunctionType),
                                 cytoolz_dict)
    
        # full API coverage should be tested elsewhere
        toolz_dict = keyfilter(lambda x: x in cytoolz_dict, toolz_dict)
        cytoolz_dict = keyfilter(lambda x: x in toolz_dict, cytoolz_dict)
    
        d = merge_with(identity, toolz_dict, cytoolz_dict)
        for key, (toolz_func, cytoolz_func) in d.items():
            if key in ['excepts', 'juxt', 'memoize', 'flip']:
                continue
            try:
                # function
>               toolz_spec = inspect.getargspec(toolz_func)
E               AttributeError: module 'inspect' has no attribute 'getargspec'

cytoolz/tests/test_embedded_sigs.py:39: AttributeError
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -14,7 +14,18 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "pypy-3.6", "pypy-3.7"]
+        python-version:
+          - "3.5"
+          - "3.6"
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
+          - "3.11.0-alpha - 3.11.0"
+          - "pypy-3.6"
+          - "pypy-3.7"
+          - "pypy-3.8"
+          - "pypy-3.9"
     steps:
       - name: Checkout
         uses: actions/checkout@v2

--- a/cytoolz/tests/test_embedded_sigs.py
+++ b/cytoolz/tests/test_embedded_sigs.py
@@ -1,7 +1,7 @@
 import inspect
 import cytoolz
 
-from types import BuiltinFunctionType
+from types import BuiltinFunctionType, FunctionType
 from cytoolz import curry, identity, keyfilter, valfilter, merge_with
 from dev_skip_test import dev_skip_test
 
@@ -30,36 +30,38 @@ def test_class_sigs():
     toolz_dict = keyfilter(lambda x: x in cytoolz_dict, toolz_dict)
     cytoolz_dict = keyfilter(lambda x: x in toolz_dict, cytoolz_dict)
 
+    class wrap:
+        """e.g., allow `factory=<class 'dict'>` to instead be `factory=dict` in signature"""
+        def __init__(self, obj):
+            self.obj = obj
+
+        def __repr__(self):
+            return getattr(self.obj, '__name__', repr(self.obj))
+
     d = merge_with(identity, toolz_dict, cytoolz_dict)
     for key, (toolz_func, cytoolz_func) in d.items():
-        if key in ['excepts', 'juxt', 'memoize', 'flip']:
-            continue
-        try:
+        if isinstance(toolz_func, FunctionType):
             # function
-            toolz_spec = inspect.getargspec(toolz_func)
-        except TypeError:
-            try:
-                # curried or partial object
-                toolz_spec = inspect.getargspec(toolz_func.func)
-            except (TypeError, AttributeError):
-                # class
-                toolz_spec = inspect.getargspec(toolz_func.__init__)
-
-        # For Cython < 0.25
-        toolz_sig = toolz_func.__name__ + inspect.formatargspec(*toolz_spec)
-        doc = cytoolz_func.__doc__
-        # For Cython >= 0.25
-        toolz_sig_alt = toolz_func.__name__ + inspect.formatargspec(
-            *toolz_spec,
-            **{'formatvalue': lambda x: '=' + getattr(x, '__name__', repr(x))}
+            toolz_spec = inspect.signature(toolz_func)
+        elif isinstance(toolz_func, toolz.curry):
+            # curried object
+            toolz_spec = inspect.signature(toolz_func.func)
+        else:
+            # class
+            toolz_spec = inspect.signature(toolz_func.__init__)
+        toolz_spec = toolz_spec.replace(
+            parameters=[
+                v.replace(default=wrap(v.default))
+                if v.default is not inspect._empty
+                else v
+                for v in toolz_spec.parameters.values()
+            ]
         )
         # Hmm, Cython is showing str as unicode, such as `default=u'__no__default__'`
-        toolz_sig_alt2 = toolz_func.__name__ + inspect.formatargspec(
-            *toolz_spec,
-            **{'formatvalue': lambda x: '=u' + getattr(x, '__name__', repr(x))}
-        )
-        doc_alt = doc.replace('Py_ssize_t ', '')
-        if not (toolz_sig in doc or toolz_sig_alt in doc_alt or toolz_sig_alt2 in doc_alt):
+        doc = cytoolz_func.__doc__
+        doc_alt = doc.replace('Py_ssize_t ', '').replace("=u'", "='")
+        toolz_sig = toolz_func.__name__ +  str(toolz_spec)
+        if not (toolz_sig in doc or toolz_sig in doc_alt):
             message = ('cytoolz.%s does not have correct function signature.'
                        '\n\nExpected: %s'
                        '\n\nDocstring in cytoolz is:\n%s'

